presto-main/src/main/java/io/prestosql/testing/MaterializedResult.java

```diff
@@ -72,6 +72,7 @@
 import static io.prestosql.spi.type.RealType.REAL;
 import static io.prestosql.spi.type.SmallintType.SMALLINT;
 import static io.prestosql.spi.type.StandardTypes.ARRAY;
+import static io.prestosql.spi.type.StandardTypes.JSON;
```

Comment:
The comparison below needs to be against `JsonType.JSON`

Reply:
JsonType is not visible. Cay you please elaborate ?